sdk/communication/Azure.Communication.CallAutomation/tests/EventCatcher/RecordedEventListener.cs

```diff
@@ -171,8 +171,9 @@ private async Task StopProcessors()
         private static string GetEventSessionFilePath(string testSessionFilePath)
         {
             // replace SessionRecords with directory name of event recordings - it'll be easier to correlate
-            var result = testSessionFilePath.Replace("SessionRecords", RecordingLocation);
-            return result;
+            var relativeEventSessionFilePath = testSessionFilePath.Replace("SessionRecords", RecordingLocation);
+
+            return Path.Combine(TestEnvironment.RepositoryRoot, relativeEventSessionFilePath);
```

Comment:
I had to update the logic to build the event session file path. It uses the original `SessionRecords` folder path as a base, but it has now been moved to the `.assets` folder.